external-providers/golang-external-provider/pkg/golang/provider.go

```diff
@@ -17,6 +17,8 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// TODO(shawn-hurley): Pipe the logger through
+// Determine how and where external providers will
```

Comment:
will what?

Reply:
Honestly don't remember  👍🏻 

Will try and update on next pass.